### PR TITLE
Added spanish

### PR DESCRIPTION
**Pull request by Hackmastr**

### CODE REVIEW

File: resource/source-python/translations/map_cycle/popups.ini [hunk 1]
```diff
@@ -1,63 +1,79 @@
...
 
 [postfix_nominated]
 en=" (+{nominations})"
 ru=" (+{nominations})"
+es=" (+{nominations})"
 
 [likes]
 en=" ♡{likes}"
 ru=" ♡{likes}"
+es=" ♡{me gusta}"
```
Comment (KirillMysnik) [inline resource/source-python/translations/map_cycle/popups.ini]: This should be

``` ini
es=" ♡{likes}"
```

This is a token, it doesn't need translating

File: resource/source-python/translations/map_cycle/strings.ini [hunk 2]
```diff
@@ -1,111 +1,139 @@
...
+es="{color_default}Voto pendiente"
 
 [error_disabled]
 en="{color_error}This command is disabled"
 ru="{color_error}Эта команда отключена"
+es="{color_error}Este comando está inhablitado"
 
 [timeleft_last_round]
-en="{color_default}This is the last round"
-ru="{color_default}Это последний раунд"
```
Comment (KirillMysnik) [inline resource/source-python/translations/map_cycle/strings.ini:43]: I'm not quite sure - why GitHub highlights these 2 lines as being replaced?..

**Review by KirillMysnik — CHANGES_REQUESTED** [review]

File: resource/source-python/translations/map_cycle/strings.ini (hunk 2 above)
Comment (Hackmastr) [reply resource/source-python/translations/map_cycle/strings.ini:43]: I've added an exclamation mark(!) to the end, that's why.

File: resource/source-python/translations/map_cycle/popups.ini (hunk 1 above)
Comment (Hackmastr) [inline resource/source-python/translations/map_cycle/popups.ini]: What does that mean? like in spanish is not the same word

Comment (Hackmastr) [reply resource/source-python/translations/map_cycle/popups.ini]: Right! Sorry.

File: resource/source-python/translations/map_cycle/strings.ini (hunk 2 above)
Comment (KirillMysnik) [reply resource/source-python/translations/map_cycle/strings.ini:43]: I'd keep it without exclamation marks. But whatever. Thank you!

**Comment by KirillMysnik** [comment]
Thank you!

**Review by KirillMysnik — APPROVED** [review]

**Pull request merged**